app/assets/javascripts/discourse/app/components/summary-box.hbs

```diff
@@ -50,12 +50,15 @@
           <div class="summarized-on">
             <p>
               {{i18n "summary.summarized_on" date=this.summarizedOn}}
-              <span>
-                {{d-icon "info-circle"}}
-                <DTooltip @placement="top-end">
+
+              <DTooltip @options={{hash allowedPlacements=(array "top-end")}}>
```

Comment:
Could the individual options be explicitly defined as d-tooltip args instead? (for better DX)